stdlib/math.rn

```diff
@@ -1,21 +1,22 @@
 # The Math class provides all types of math functionalites for radon language.
 
-static const PI = 3.14159265358979323846
+const PI = 3.14159265358979323846
 
-static fun sqrt(number) -> number ^ 0.5
-static fun pow(base, power) -> base ^ power
```

Comment:
What happened to `static` keyword here?

Reply:
It doesn't make sense here as it is not a class